pkgs/servers/nosql/mongodb/mongodb.nix

```diff
@@ -44,7 +68,9 @@ in stdenv.mkDerivation rec {
     inherit sha256;
   };
 
-  nativeBuildInputs = [ variants.scons ];
+  nativeBuildInputs = [ variants.scons ]
+    ++ lib.optionals (stdenv.isLinux && versionAtLeast version "4.4") [ lzma ];
```

Comment:
The manual don't state LZMA being platform-specific so please pin it to the version only.

Reply:
```suggestion
    ++ lib.optionals (versionAtLeast version "4.4") [ xz ];
```